addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -818,6 +818,18 @@ const Wysiwyg = Widget.extend({
             observerData.observer.disconnect();
         }
     },
+    /**
+     * Activates the link tool for the image.
+     */
+    toggleLinkOnImage() {
+        const linkButtonEl = document.querySelector('we-button[data-set-link]');
+        if (!linkButtonEl.classList.contains('active')) {
+            linkButtonEl.click();
+        }
+        setTimeout(function () { // Wait for the URL input to appear.
+            document.querySelector('we-input[data-set-url] input').focus();
+        }, 300);
```

Comment:
Does the URL need to be focused when the link is removed ?

Really 300 ? Is there another way to find out when the `click()` effect is completed ?